lib/tasks/find_verified_users_not_in_csv.rake

```diff
@@ -0,0 +1,137 @@
+namespace :users do
+  desc 'Finds all verified users NOT present in a CSV file'
```

Comment:
this seems like a medium complex task, how long do we expect it to live for in the codebase? if it's gonna be around for a bit should we add specs?

Reply:
Yeah, I only need it for this one ticket, but the _idea_ ("find me users who don't have an event in this list of events I exported from Cloudwatch") seems generally useful. I'll write up some specs.